src/qt/sendcoinsdialog.cpp

```diff
@@ -373,7 +393,12 @@ void SendCoinsDialog::on_sendButton_clicked()
         questionString = questionString.arg("<br /><br />" + formatted.at(0));
     }
 
-    SendConfirmationDialog confirmationDialog(tr("Confirm send coins"), questionString, informative_text, detailed_text, SEND_CONFIRM_DELAY, this);
+    const QString confirmation = model->privateKeysDisabled() ? tr("Confirm transaction proposal") : tr("Confirm send coins");
+    QString confirmButtonText = tr("Send");
```

Comment:
9808f7867d9c557ba4bc253104f47fbd7f02b668

Use ternary like above or just one `if (model->privateKeysDisabled())` for these.